repair/style.css

```diff
@@ -111,6 +111,19 @@
 .home-repair-col1 button:hover:after {
     opacity: 1;
   }
+  .home-repair-col button{
```

Comment:
Why use the same CSS classes twice? Can we not combine both under a single class (`.home-repair-col button`)?